clicConfig/clicReconstruction.xml

```diff
@@ -1647,6 +1647,7 @@
     <parameter name="PrintEventNumber" type="int" value="1"/> <!-- 0 for not printing event number, n for printing every n events -->
     <parameter name="MagneticField" type="float" value="4.0"/> <!-- ILC and CLIC detectors have different values -->
     <!-- jet clustering parameters -->
+    <parameter name="JetClustering.PrimaryVertexCollectionName" type="string" value="pVx" /> <!-- Needed for post 00.06.09 -->
```

Comment:
@eleogran In fact this makes things forward compatible, please add the line also for the fccRecoConfig.